### ISSUE TITLE
Definition of loss in pre-train phase 

### ISSUE THREAD

**Issue by leihouyeung** [rsinghlab/scNODE#1]
Hi, thanks for your great work. I'm just wondering if there is misalignment between the definition of loss in pre-train phase [(here)](https://github.com/rsinghlab/scNODE/blob/9b69b65f46f4e5b853167ede1c6609d91089a831/optim/running.py#L108) and the Equation 2 in the manuscript? I think there should be a KL loss that author missed. Currently, MSE loss is used only.

**Comment by JQZ-Brown** [comment]
Hi, thank you for your question.

In all experiments, we set the KL regularization coefficient $\lambda=0$ in pertaining. Because we have tested various $\lambda$ values and found $\lambda=0$ leads to the best performance. We hypothesize that this is because cell latent representations do not necessarily follow the prior normal distribution so that the KL penalty may introduce biases in the latent representation learning. This explanation has now been added to Supplementary Section 5.

Overall, the VAE pretraining should contain KL. However, we found that using only MSE in our datasets produces better results in our datasets. I think this is data-dependent. I will add the KL term back and update the codes.
